src/components/ClaimCard.svelte

```diff
@@ -19,7 +19,7 @@ $: items = $itemsByPolicyId[policyId] || []
 $: item = items.find(i => i.id = claimItem.item_id) || {}
 $: msAgo = now - Date.parse(claimItem.updated_at)
 $: daysAgo = msAgo > 0 ? Math.floor(msAgo/day) : '-'
-$: state = getState(claim) || {}
+$: state = claim.status && getState(claim.status) || {}
```

Comment:
It's not obvious whether this will return `undefined` or `{}` if the claim has no status. Why the `claim.status` check before the call to `getState()`? I think it will handle undefined just fine.

Reply:
before I added it, getState was being called with undefined causing an error. An alternative to this would be passing it a default state if claim.status was undefined.

Reply:
I think with your corresponding change to `getState()` it will no longer throw an error. However, if you don't mind wrapping this check in parentheses to make it clear we'll always get an object back (not undefined) for `state`, that's also fine:
```suggestion
$: state = (claim.status && getState(claim.status)) || {}
```

Reply:
You are right about it no longer throwing an error since it only fires off when claim.status changes.